static/app/views/alerts/rules/issue/details/textRule.tsx

```diff
@@ -21,26 +21,18 @@ export function TextCondition({
 }) {
   const organization = useOrganization();
 
-  if (CHANGE_ALERT_CONDITION_IDS.includes(condition.id)) {
+  if (
+    condition.id === IssueAlertConditionType.EVENT_FREQUENCY_PERCENT ||
+    condition.id === IssueAlertConditionType.EVENT_FREQUENCY ||
+    condition.id === IssueAlertConditionType.EVENT_UNIQUE_USER_FREQUENCY
+  ) {
+    const subject = CHANGE_ALERT_PLACEHOLDERS_LABELS[condition.id];
     if (condition.comparisonType === AlertRuleComparisonType.PERCENT) {
-      if (condition.id === IssueAlertConditionType.EVENT_FREQUENCY_PERCENT) {
-        return (
-          <Fragment>
-            {t(
-              // Double %% escapes
-              'Percent of sessions affected by an issue is %s%% higher in %s compared to %s ago',
-              condition.value,
-              condition.interval,
-              condition.comparisonInterval
-            )}
-          </Fragment>
-        );
-      }
       return (
         <Fragment>
           {t(
```

Comment:
Is this even translatable? `subject` is an english phrase, so it would end up being two different languages mixed up

Reply:
it probably is not